website/docs/api/cli.md

```diff
@@ -175,7 +175,7 @@ $ python -m spacy init fill-config [base_path] [output_file] [--diff]
 | Name                   | Description                                                                                                                                                                          |
 | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `base_path`            | Path to base config to fill, e.g. generated by the [quickstart widget](/usage/training#quickstart). ~~Path (positional)~~                                                            |
-| `output_file`          | Path to output `.cfg` file. If not set, the config is written to stdout so you can pipe it forward to a file. ~~Path (positional)~~                                                  |
+| `output_file`          | Path to output `.cfg` file or "-" to write to stdout so you can pipe it forward to a file. ~~Path (positional)~~                                                                     |
```

Comment:
Does this still write to stdout if not set?

Also I would remove "forward" here.

Reply:
I've clarified that "-" is the default